### PR TITLE
Build out Grove Playlist Manager functionality

### PR DESCRIPTION
**Pull request by erinnachen**
- On the grove playlist manager, shows all students in a grove
- Can now search by name
- When a student is created, it also creates an associated playlist.

@jedschneider I saw that you built out resource view templates, but I didn't use it because I didn't need everything that was included in the template.

### CODE REVIEW

File: app/controllers/admin/students_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin::StudentsController < ApplicationController
+  def index
```
Comment (jedschneider) [inline app/controllers/admin/students_controller.rb]: inherit from the ResourceController and specify the scope in the collection.

**Comment by jedschneider** [comment]
move the index file to an `_index.html.erb` file and reference collection, not students.

File: app/controllers/admin/students_controller.rb (hunk 1 above)
Comment (jedschneider) [inline app/controllers/admin/students_controller.rb]: here https://github.com/rootselementary/check_in_tool_rails/blob/4cc2a99b7162d56c5ea8e04c7b40a536bb2b3964/app/controllers/admin/teachers_controller.rb#L20

**Comment by jedschneider** [comment]
i disagree with your original comment, so just pointing you towards how to do that with the resource controller. but I do suspect you will need more custom-ness than the resource controller is going to give you in this admin resource namespace. So, you can probably carry on as you are.

**Comment by erinnachen** [comment]
Cool. I just hadn't realized if I could override the template. I'll change things to inherit from resources.

File: app/models/users/student.rb [hunk 2]
```diff
@@ -12,4 +14,11 @@ def after_sign_in_path
   def self.is_student?(data)
     Student.where(email: data["email"]).first
   end
+
+  private
+
+  def create_student_playlist
+    student = Student.last
```
Comment (NickyBobby) [inline app/models/users/student.rb:21]: Instead of using Student.last to get the last created student, would it be better to just put `self` on line 22?

**Comment by NickyBobby** [comment]
I pulled down the branch and played around with it a little bit. Seems to be working well. This PR is GTM!

**Pull request merged**